festim/exports/derived_quantities/surface_flux.py

```diff
@@ -3,24 +3,60 @@
 
 
 class SurfaceFlux(SurfaceQuantity):
-    def __init__(self, field, surface) -> None:
-        """
+    """
+    Computes the surface flux of a field at a given surface
+
+    Args:
+        field (str, int):  the field ("solute", 0, 1, "T", "retention")
+        surface (int): the surface id
+
+    Attribtutes
+        field (str, int):  the field ("solute", 0, 1, "T", "retention")
+        surface (int): the surface id
+        show_units (bool): show the units in the title in the derived quantities
+            file
+        title (str): the title of the derived quantity
+        function (dolfin.function.function.Function): the solution function of
+            the hydrogen solute field
+
+    Notes:
         Object to compute the flux J of a field u through a surface
         J = integral(-prop * grad(u) . n ds)
         where prop is the property of the field (D, thermal conductivity, etc)
         u is the field
         n is the normal vector of the surface
         ds is the surface measure.
 
-        Note: this will probably won't work correctly for axisymmetric meshes.
-        Use only with cartesian coordinates.
+        units are in H/m2/s in 1D, H/m/s in 2D and H/s in 3D domains for hydrogen
+        concentration and W/m2 in 1D, W/m in 2D and W in 3D domains for temperature
 
-        Args:
-            field (str, int):  the field ("solute", 0, 1, "T", "retention")
-            surface (int): the surface id
-        """
+    """
+
+    def __init__(self, field, surface, show_units=False) -> None:
```

Comment:
```suggestion
    def __init__(self, field, surface) -> None:
```

There is no reason for `show_units` to be an argument here. Proof is, you don't use it.